cfme/tests/services/test_service_catalogs.py

```diff
@@ -27,11 +27,11 @@
 def pytest_generate_tests(metafunc):
     # Filter out providers without provisioning data or hosts defined
     argnames, argvalues, idlist = testgen.infra_providers(metafunc,
-        required_fields=[
-            ['provisioning', 'template'],
-            ['provisioning', 'host'],
-            ['provisioning', 'datastore']
-        ])
+                                                          required_fields=[
```

Comment:
oh, I mistakenly aligned this code using pycharm's feature. 
I just forgot that not all people here use pycharm .